gammapy/irf/energy_dispersion.py

```diff
@@ -116,47 +116,37 @@ def pdf_in_safe_range(self, lo_threshold, hi_threshold):
         return data
 
     @classmethod
-    def from_gauss(cls, e_true, e_reco, sigma=0.2, pdf_threshold=1e-6):
+    def from_gauss(cls, e_true, e_reco, sigma, bias, pdf_threshold=1e-6):
         """Create Gaussian `EnergyDispersion` matrix.
 
-        The output matrix will be Gaussian in log(e_true / e_reco)
-
-        TODO: extend to have a vector of bias various true energies.
-        TODO: extend to have vector of  resolution for various true energies.
-        TODO: give formula: Gaussian in log(e_reco)
-        TODO: add option to add poisson noise
+        Calls :func:`~gammapy.irf.EnergyDispersion2D.from_gauss`
 
         Parameters
         ----------
         e_true : `~astropy.units.Quantity`, `~gammapy.utils.nddata.BinnedDataAxis`
             Bin edges of true energy axis
         e_reco : `~astropy.units.Quantity`, `~gammapy.utils.nddata.BinnedDataAxis`
             Bin edges of reconstructed energy axis
-        sigma : float, optional
+        bias : float or `~numpy.ndarray`
+            Center of Gaussian energy dispersion, bias
+        sigma : float or `~numpy.ndarray`
             RMS width of Gaussian energy dispersion, resolution
         pdf_threshold : float, optional
             Zero suppression threshold
         """
-        from scipy.special import erf
-
-        e_reco = EnergyBounds(e_reco)
-        e_true = EnergyBounds(e_true)
-
-        # erf does not work with Quantities
-        reco = e_reco.to('TeV').value
-        true = e_true.log_centers.to('TeV').value
-        migra_min = np.log10(reco[:-1] / true[:, np.newaxis])
-        migra_max = np.log10(reco[1:] / true[:, np.newaxis])
-
-        pdf = .5 * (erf(migra_max / (np.sqrt(2.) * sigma))
-                    - erf(migra_min / (np.sqrt(2.) * sigma)))
-
-        pdf[np.where(pdf < pdf_threshold)] = 0
-
-        e_lo, e_hi = (e_true[:-1], e_true[1:])
-        ereco_lo, ereco_hi = (e_reco[:-1], e_reco[1:])
-        return cls(e_true_lo=e_lo, e_true_hi=e_hi,
-                   e_reco_lo=ereco_lo, e_reco_hi=ereco_hi, data=pdf)
+        # migra axis
+        migra = np.linspace(1./3, 3, 200)
+        
+        # offset axis
+        offset = Quantity([0,1,2], 'deg')
```

Comment:
I don't understand why you create two (or three?) offset bins, and then later only slice out the first one.
Would it be possible to only create one bin here if only one is needed?
Or even more radical: would it make sense to switch things around and have EnergyDispersion2D.from_gauss call EnergyDispersion.from_gauss and then to np.repeat along the offset axis?
This should simplify the combined code of the two methods, and mean that you don't have to talk about offset here in EnergyDispersion.from_gauss, no?

Reply:
> Or even more radical: would it make sense to switch things around and have EnergyDispersion2D.from_gauss call EnergyDispersion.from_gauss and then to np.repeat along the offset axis?

That's a good point, I will make the changes accordingly

Reply:
Since this will need to be changed anyways I don't touch this